config-linux.md

```diff
@@ -460,25 +460,34 @@ The following parameters can be specified to set up the controller:
 **`intelRdt`** (object, OPTIONAL) represents the [Intel Resource Director Technology][intel-rdt-cat-kernel-interface].
     If `intelRdt` is set, the runtime MUST write the container process ID to the `<container-id>/tasks` file in a mounted `resctrl` pseudo-filesystem, using the container ID from [`start`](runtime.md#start) and creating the `<container-id>` directory if necessary.
     If no mounted `resctrl` pseudo-filesystem is available in the [runtime mount namespace](glossary.md#runtime-namespace), the runtime MUST [generate an error](runtime.md#errors).
-
```

Comment:
nit: I liked having the not-set sentence in its own paragraph to make it even more obvious that the no-mounted line belonged to the if-set case.  But I don't care all that much, and am fine with this PR landing with this paragraph-splitting-newline removed.

Reply:
@wking 
The reason why I delete this blank line is that: I found a nit in [original master branch code](https://github.com/opencontainers/runtime-spec/blob/master/config-linux.md): two or more spaces don't work at the end of "no-mounted" sentence for newline, and the "no-set" sentence is in code syntax highlighting style.

I could fix this issue to make "not-set" sentence in its own paragraph:
(without four spaces at the beginning of "not-set" sentence)
```
    If no mounted `resctrl` pseudo-filesystem is available in the [runtime mount namespace](glossary.md#runtime-namespace), the runtime MUST [generate an error](runtime.md#errors).

If `intelRdt` is not set, the runtime MUST NOT manipulate any `resctrl` pseudo-filesystems.
```